src/server/pfs/server/driver.go

```diff
@@ -2538,6 +2538,43 @@ func (d *driver) getFile(pachClient *client.APIClient, file *pfs.File, offset in
 	return grpcutil.NewStreamingBytesReader(getBlocksClient, nil), nil
 }
 
+func (d *driver) filesFromByteStream(pachClient *client.APIClient, objReader io.Reader, file *pfs.File, offset, size int64) (io.ReadCloser, error) {
```

Comment:
@brycemcanally mainly wanted you to look at this and see that it looks like what you expected. As we discussed, it reads out and buffers bytes from what we get back from the `getBlocks` and `getObjects` calls in `driver.getFile`, generates `GetFileResponse` messages, and then writes those to a separate reader that get used by the client to receive the stream of `GetFileResponse` messages.

If this looks good so far, I will add the last piece which is to properly use the file size determined in the driver rather than the request size to break out the bytes we get into discreet messages.